Xamarin.Forms.Core/Switch.cs

```diff
@@ -17,6 +17,8 @@ public class Switch : View, IElementConfiguration<Switch>
 
 		public static readonly BindableProperty ThumbColorProperty = BindableProperty.Create(nameof(ThumbColor), typeof(Color), typeof(Switch), Color.Default);
 
+		public static readonly BindableProperty IsReadOnlyProperty = BindableProperty.Create(nameof(IsReadOnly), typeof(bool), typeof(Switch), false);
```

Comment:
I'd prefer having the property named `ReadOnly`

Reply:
> how is this different from `IsEnabled` ?

In visual, it is not toggleable but also does not have the disabled appearance.

The issue author and @PureWeen can probably describe their use cases for it better, but my reasoning has been that sometimes I want to show that a value is there, but not allow it to be toggled currently. 

Disabled look doesn't feel right in those scenarios, but an un-Switchable Switch with a different thumb color does.





> I'd prefer having the property named `ReadOnly`

I agree, but I just followed the naming conventions of IsVisible, IsEnabled, and the Entry IsReadOnly.

Reply:
> does not have the disabled appearance

this might be visually confusing... "why is it not working"

Reply:
> property named `ReadOnly`

Is this property used elsewhere? ie: for editor/entry? What is it called?

Reply:
> > does not have the disabled appearance
> 
> this might be visually confusing... "why is it not working"

The use case I had in mind at least is a "locked" switch. It has a different thumb color, indicating is locked, but, doesn't look disabled so that it appears to still apply, even if I am not allowed to change the value.

I am not sure what other people were intending it for, but I have wanted it enough times that I wrote the PR when I saw it as an issue.


> > property named `ReadOnly`
> 
> Is this property used elsewhere? ie: for editor/entry? What is it called?

Entry/Editor appears to have gone with [IsReadOnly](https://github.com/xamarin/Xamarin.Forms/blob/master/Xamarin.Forms.Core/InputView.cs#L17).